app-misc/cadubi: version bump (1.3.4)

mm1ke opened the pull request:
Hi,

Simple version bump for `app-misc/cadubi`.
This fixes the `HOMEPAGE` to use github, uses `sed` to fix the `help.txt` file and also installs the man page of the application.

Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Michael Mair-Keimberger <mmk@levelnine.at>

ConiKost commented on app-misc/cadubi/cadubi-1.3.3.ebuild:
```diff
@@ -0,0 +1,28 @@
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+
+DEPEND="dev-lang/perl
```
Shouldn't be this ```RDEPEND```? Based on upstreams repo, this seems only normal perl scripts without any compilation involved?

ConiKost reviewed (changes requested):
Could you also bumped directly to ```1.3.4```? Seems via tags available: https://github.com/statico/cadubi/tags

mm1ke commented:
> Could you also bumped directly to `1.3.4`? Seems via tags available: https://github.com/statico/cadubi/tags

Updated the ebuild to use `1.3.4`. Also changed it to `RDEPEND`, seems to be a simple perl script for me too.
